packages/webdriverio/src/commands/element/clearElement.js

```diff
@@ -7,19 +7,19 @@
  * <example>
     :clearElement.js
     it('should demonstrate the clearElement command', function () {
-        var input = $('.input')
-        input.setValue('test123')
-        console.log(input.getValue()) // returns 'test123'
-        input.clearElement()
-        // or
-        browser.clearElement('.input')
-        var value = browser.getValue('.input')
+        let elem = browser.$('.input')
```

Comment:
How about using the shortener:
```js
let elem = $('.input')
```

Reply:
Sure.  Even though it is more to type, I regularly find myself using ```browser.$()``` because it **feels** more readable.